.travis.yml

```diff
@@ -1,59 +1,64 @@
 language: ruby
+sudo: required
+services:
+  - docker
 cache:
   bundler: true
   yarn: true
   directories:
-    - node_modules
-    - public/assets
-    - public/packs-test
-    - tmp/cache/babel-loader
+  - node_modules
+  - public/assets
+  - public/packs-test
+  - tmp/cache/babel-loader
 dist: trusty
 sudo: false
 branches:
   only:
-    - master
-
+  - master
+  - /^v\d+\.\d+(\.\d+)?(\S*)?(-\S*)?$/
 notifications:
   email: false
-
 env:
   global:
-    - LOCAL_DOMAIN=cb6e6126.ngrok.io
-    - LOCAL_HTTPS=true
-    - RAILS_ENV=test
-    - NOKOGIRI_USE_SYSTEM_LIBRARIES=true
-    - PARALLEL_TEST_PROCESSORS=2
-
+  - LOCAL_DOMAIN=cb6e6126.ngrok.io
+  - LOCAL_HTTPS=true
+  - RAILS_ENV=test
+  - NOKOGIRI_USE_SYSTEM_LIBRARIES=true
+  - PARALLEL_TEST_PROCESSORS=2
+  - DOCKER_HUB_USERNAME=gargron
+before_install:
+  - curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
+  - sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
+  - sudo apt-get update
+  - sudo apt-get -y install docker-ce
 addons:
   postgresql: 9.4
   apt:
     sources:
-      - trusty-media
-      - sourceline: deb https://dl.yarnpkg.com/debian/ stable main
-        key_url: https://dl.yarnpkg.com/debian/pubkey.gpg
+    - trusty-media
+    - sourceline: deb https://dl.yarnpkg.com/debian/ stable main
+      key_url: https://dl.yarnpkg.com/debian/pubkey.gpg
     packages:
-      - ffmpeg
-      - libicu-dev
-      - libprotobuf-dev
-      - protobuf-compiler
-      - yarn
-
+    - ffmpeg
+    - libicu-dev
+    - libprotobuf-dev
+    - protobuf-compiler
+    - yarn
 rvm:
-  - 2.4.2
-  - 2.5.0
-
+- 2.4.2
+- 2.5.0
 services:
-  - redis-server
-
+- redis-server
 install:
-  - nvm install
-  - bundle install --path=vendor/bundle --without development production --retry=3 --jobs=16
-  - yarn install
-
+- nvm install
+- bundle install --path=vendor/bundle --without development production --retry=3 --jobs=16
+- yarn install
 before_script:
-  - ./bin/rails parallel:create parallel:load_schema parallel:prepare assets:precompile
-
+- "./bin/rails parallel:create parallel:load_schema parallel:prepare assets:precompile"
```

Comment:
Redundant quotes.

Reply:
This went through Travis' own linter (which is why some parts of the file have changed) and I'd rather like to leave it that way.